### PR TITLE
Added Communication Section to Teamwork

### PR DESCRIPTION
**Pull request by aielaayaz**
Submitted PR for contribution

### CODE REVIEW

File: Topics/Teamwork.md [hunk 1]
```diff
@@ -1,5 +1,22 @@
 ## Resources for Teamwork
 
+## Communication
```
Comment (JerryZhao1025) [inline Topics/Teamwork.md:3]: Overall, the paragraph provides a good overview of the importance of communication and conflict resolution in software engineering teams.

Comment (JerryZhao1025) [inline Topics/Teamwork.md:3]: You can provide more specific examples or case studies of how **effective communication and conflict resolution** have **positively impacted** implications of these concepts.

Comment (aielaayaz) [reply Topics/Teamwork.md:3]: Just did so in the latest commit!

Comment (aielaayaz) [reply Topics/Teamwork.md:3]: Thank you!

File: Topics/Teamwork.md [hunk 2]
```diff
@@ -1,5 +1,25 @@
...
+
+[How to avoid misunderstandings in the workplace](https://www.roberthalf.cn/en/management-advice/team/how-avoid-misunderstandings-workplace)
+
+Communication is essential for positive collaboration. In software engineering different problems can arise and effective communication is key to resolving them quickly. By discussing issues openly and honestly, team members can brainstorm solutions and use their collective expertise to work together and resolve issues in a timely manner. Effective communication, therefore, leads to the development of high-quality software products.
+
+[7 tips for more effective communication in the workplace](https://asana.com/resources/effective-communication-workplace)
```
Comment (JerryZhao1025) [inline Topics/Teamwork.md:12]: This is a good one. Key components of effective communication may also include:
A. Active listening
B. Clear and concise messaging
C. Open and honest feedback
D. Timely communication

File: Topics/Teamwork.md [hunk 3]
```diff
@@ -1,5 +1,25 @@
 ## Resources for Teamwork
 
+## Communication
+
+
```
Comment (JerryZhao1025) [inline Topics/Teamwork.md:5]: You can also include the Challenges to effective communication:
like Language and cultural differences, Technology barriers, Time zone differences and/or Personal biases and emotions

**Pull request merged**